Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml

```
@@ -319,3 +319,26 @@
         - id: OxygenTankFilled
         - id: ClothingOuterHardsuitWizard
         - id: ClothingMaskBreath
+
+#Cybersun Armoring Corporation
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)